pkg/operator/ceph/csi/csi.go

```diff
@@ -79,6 +80,9 @@ func (r *ReconcileCSI) setParams() error {
 		CSIParam.ForceCephFSKernelClient = "true"
 	}
 
+	opcontroller.SetRevisionHistoryLimit(r.opConfig.Parameters)
```

Comment:
This is already called here (below) and is recorded globally. Do we need to call this again in the CSI controller? 

https://github.com/rook/rook/blob/ab8fd90aa642ea670fd8786e4b9bbb04b6beb8e2/pkg/operator/ceph/controller.go#L136

Reply:
Originally, I thought, `SetRevisionLimit` might not have been called globally at this point. removed this call now.